src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -195,7 +195,9 @@ protected void handleUserException(Exception e) {
         if(printStackTraceOnUserExceptions()) {
             e.printStackTrace();
         } else {
-            System.err.println("Use -D" + STACK_TRACE_ON_USER_EXCEPTION_PROPERTY + "to print the stack trace.");
+            System.err.println("Use the system property "
+                            + STACK_TRACE_ON_USER_EXCEPTION_PROPERTY
+                            + " (--javaOptions '-DGATK_STACKTRACE_ON_USER_EXCEPTION=true') to print the stack trace.");
```

Comment:
@cmnbroad I obviously didn't pay enough attention here because I missed the fact that the property name is hardcoded now in part of the message, we should have it rely on the constant instead.